New feature : Issue template (#3)
Libra-foundation/.github: ISSUE_TEMPLATE/ISSUE_TEMPLATE.md

# Prerequisites

> Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

Reviewer: I think we could move this to the bottom and tell them to let it. Sometimes you know you can't check a box just because you don't have that information. For example, I found a bug in a web app, so I report it to the front repo. However, the bug might come from the server I don't know I don't have the knowledge for this, then I might would like to tell that I don't have this information.

- [ ] I am running the latest version
- [ ] I checked the documentation and found no answer
- [ ] I checked to make sure that this issue has not already been filed
- [ ] I'm reporting the issue to the correct repository (for multi-repository projects)

# Expected Behavior

Please describe the behavior you are expecting

# Current Behavior

What is the current behavior?

> # Failure Information (for bugs)

Reviewer: For what would be an issue if not for a bug, since enhancement should be in the discussion ?

Please help provide information about the failure if this is a bug. If it is not a bug, please remove the rest of this template.

## Steps to Reproduce

Please provide detailed steps for reproducing the issue.

1. step 1
2. step 2
3. you get it...

## Context

Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.

> * Firmware Version:

Reviewer: This list could be reworked. I do not see in which case we can ask for an SDK, or a Tool chain. I also don't see the use of the Firmware version, what Firmware would you want the version of ?

What is the current behavior?

> # Failure Information

Reviewer: Not sure about the goal of this section, can you please enlighten me ?

## Context

Please provide any relevant information about your setup. This is important in case the issue is not reproducible except when under certain conditions.

> * Environment version (node, npm, python... pick only the relevant ones

Reviewer: ```suggestion
* Environment version (node, npm, python... pick only the relevant ones)
```

> Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.

Reviewer: ```suggestion
Please provide any relevant information about your setup. This is important in case the issue is not reproducible except when under certain conditions.
```

> * What types of projects you work on ?
> * Which language versions are you using ?
> * Your IDE and its version ?
> * From when error occurred ?

Reviewer: ```suggestion
* Environment version (node, npm, python... pick only the relevant ones
* Platform (Linux, Windows, Mac os)
* Browser with version (if applicable)
* IDE with version (if applicable)
* Anything that might be useful to reproduce your issue
```

> ## Failure Logs

Reviewer: ```suggestion
## Logs
```

> Please include any relevant log snippets or files here.

Reviewer: ```suggestion
Please include any relevant log here. Compleate log files are prefered, remember to anonimise any personal data in it.
```

> # Prerequisites

Reviewer: ```suggestion
# Checks
```
